src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -303,6 +303,9 @@ theorem toInt_ofNat {n : Nat} (x : Nat) :
 @[simp] theorem ofInt_natCast (w n : Nat) :
   BitVec.ofInt w (n : Int) = BitVec.ofNat w n := rfl
 
+@[simp] theorem ofInt_ofNat (w n : Nat) :
+  BitVec.ofInt w (no_index (OfNat.ofNat n)) = BitVec.ofNat w n := rfl
```

Comment:
```suggestion
  BitVec.ofInt w (no_index (OfNat.ofNat n)) = BitVec.ofNat w (OfNat.ofNat n) := rfl
```
otherwise this can leave an unprotected raw nat literal.

Reply:
Nice. Thank you for your feedback. This is super helpful.